src/tool_operate.c

```diff
@@ -923,7 +923,7 @@ static CURLcode single_transfer(struct GlobalConfig *global,
              etag_from_file) {
             if (strlen(etag_from_file)) {
               char *header = NULL;
-              header = aprintf("If-None-Match: \"%s\"", etag_from_file);
+              header = aprintf("If-None-Match: %s", etag_from_file);
```

Comment:
maybe these two lines could be combined? seems pointless to assign it to NULL just the line before it gets assigned again....

Reply:
sure, I just moved the line here not changed it keep diff clearer.

Reply:
Should I add a separate commit with merge of these two lines?

Reply:
On second thoughts, let's just leave it like you've done it.